docs/resources/aws_shield_subscription.md

```diff
@@ -0,0 +1,55 @@
+---
+title: About the aws_shield_subscription Resource
+platform: aws
+---
+
+# aws\_shield\_subscription
+
+Use the `aws_shield_subscription` InSpec audit resource to test properties of a shield subscription.
```

Comment:
I think we should use `shield advanced subscription` instead of `shield subscription` in the whole document.